experimentalSetups/ankleSetup/ankleSetup_telemetry_data/bufferConfigankleSetup_test.json

```diff
@@ -0,0 +1 @@
+{"auto_save":true,"channels":[],"data_threshold":0,"description_list":["ankle_joint"],"enable_compression":false,"file_indexing":"time_since_epoch","filename":"ankleSetup_test","mat_file_version":"v7_3","n_samples":800000,"path":"./ankleSetup_telemetry_data/","save_period":0.01,"save_periodically":false,"yarp_robot_name":"ergoCubSN002S"}
```

Comment:
Please remove this file and the parent folder. thanks

Reply:
Done!